README.md

```diff
@@ -1 +1,68 @@
+>
+>**Methods** We estimate the additional number of COVID-19 cases that would have been reported had the sensitivity of the laboratory test procedure not dropped for the period 2 September to 12 October. In addition, by making comparisons between the most affected 
+local areas and comparator populations, we estimate the number of additional infections, cases, hospitalisations and deaths that could have occurred as a result of increased transmission due to the misclassification of tests.
+>
+>**Results** We estimate that around 39,000 tests may have been incorrectly classified during this period and, as a direct result of this incident, the most affected areas in the South West could have experienced between 6,000 and 34,000 additional reportable cases, with a central estimate of around 24,000 additional reportable cases. Using modelled relationships between key variables, we estimate that this central estimate could have translated to approximately 55,000 additional infections, which means that each incorrect negative test likely led to just over 2 additional infections. In those same geographical areas, our results also suggest an increased number of admissions and deaths.
+>
+>**Conclusion** The incident is likely to have had a measurable impact on cases and infections in the affected areas in the South West of England.
+
+### **Serious Untoward Incident** 
+The Serious Untoward Incident Investigation report can be found by accessing the following [link](https://www.gov.uk/government/publications/serious-untoward-incident-investigation-immensa-health-clinic-limited)
+
+## Repository Organization
+
+
+    │── data               <- Holds data files
+    │
+    ├── outputs
+    │ └── report           <- Where the Rmarkdown preprint will be outputted       
+    │ └── visuals          <- Plots and visualisations will be created within this folder
+    │
+    │── main_script.R      <- Runs the analysis and produces the preprint
+    │
+    ├── src                <- Source code for use in this project
+    │ └── example_foo.R    <- Example R file containing functions used in scripts.
+    │
+    ├── .Rproj             <- R project for analysis
+    │
+    ├── SECURITY.md        <- Security terms and conditions
+    │
+    └── README.md          <- The top-level README for developers using this project
+
+
+
+Replication
+----------
+This repository contains the code & data for the preprint publication (INSERT LINK HERE). The code is written in R, and data stored in csv format and a shape file will be called from the ONS api and stored as a geojson after the first run.

```

Comment:
insert link to medxiv preprint paper on this line

Reply:
Actioned